Source/Scene/FrameRateMonitor.js

```diff
@@ -98,7 +98,7 @@ define([
         this._pauseCount = 0;
 
         var that = this;
-        this._preRenderRemoveListener = this._scene.preRender.addEventListener(function(scene, time) {
```

Comment:
There are still a lot of other files and Sandcastle demos that reference `preRender` and `postRender`, do those need to use the update listeners instead too?

Reply:
Usage of postRender didn't need to be changed in the examples, but I modified the `preRender` events where necessary.